lib/puppet/transport/philips_hue.rb

```diff
@@ -0,0 +1,56 @@
+require 'faraday'
+require 'uri'
+require 'hocon'
+require 'hocon/config_syntax'
+require 'puppet/util/network_device/base'
```

Comment:
```
require 'hocon'
require 'hocon/config_syntax'
require 'puppet/util/network_device/base'
```

these can be removed as they are no longer used

Reply:
Removed